File: MyAnimeList/src/main/java/com/example/myanimelist/service/img/IImgStorage.java
package com.example.myanimelist.service.img;

import com.example.myanimelist.models.User;
import com.example.myanimelist.views.models.Presentation;
import javafx.scene.image.Image;

public interface IImgStorage {

    void mkdir();

    Image loadImg(User user);

    Image loadImg(Presentation present);
}

File: MyAnimeList/src/main/kotlin/com/example/myanimelist/service/img/IImgStorage.kt
package com.example.myanimelist.service.img

import com.example.myanimelist.models.User
import com.example.myanimelist.views.models.Presentation
import javafx.scene.image.Image
import java.io.File

interface IImgStorage {
    fun mkdir()
    fun cpFile(file: File, savePath: String?): Boolean
    fun loadImg(user: User): Image
    fun loadImg(present: Presentation): Image
}